client/src/components/internship/EditInternship.vue

```diff
@@ -219,8 +420,14 @@ export default defineComponent({
     },
     async getAvailablePaymentTypes() {
       try {
-        const res = await http.get('/info/payment-types');
-        this.availablePaymentTypes = res.data;
+        const paymentTypes = await loadPaymentTypes();
+        const st = 'internship.form.paymentType.';
+        if (paymentTypes.length > 0) {
+          // eslint-disable-next-line no-restricted-syntax
+          for (const pt of paymentTypes) {
+            this.availablePaymentTypes.push(`${this.$t(st + pt.replace(/\s/g, ''))}`);
```

Comment:
Das hier müsste denke ich kein Template String sein :)
```suggestion
            this.availablePaymentTypes.push(this.$t(st + pt.replace(/\s/g, '')));
```